classes/Renderer/Header.php

```diff
@@ -0,0 +1,307 @@
+<?php
+
+namespace MySociety\TheyWorkForYou\Renderer;
+
+/**
+ * Template Header Renderer
+ *
+ * Prepares variables for inclusion in a template header.
+ */
+
+class Header
+{
+
+    public $top_highlight;
+    public $nav_highlights;
+
+    /**
+     * Generate header data
+     *
+     * @param array  $data     An associative array of data to be made available to the template.
+     */
+
+    public function get_data($data = array())
```

Comment:
Similarly to footer, none of header needs anything passed in by $data, so it could have its own $data, and then the caller can e.g. array_merge it. Dunno, not much in it is there, perhaps leave it.